src/test/java/seedu/duke/search/SearchTest.java

```diff
@@ -0,0 +1,37 @@
+package seedu.duke.search;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.command.SearchModuleCodeCommand;
+import seedu.duke.command.SearchModuleNameCommand;
+import seedu.duke.model.Module;
+
+public class SearchTest {
+    @Test
+    void filterModuleByCode_validInput_expectCorrectNumberOfFilteredModule() {
+        // String inpuString = "cs1010";
+        String inpuString = "GEA1000";
```

Comment:
`inputString` instead of `inpuString` perhaps?

Reply:
oh yes spelling mistake. will change